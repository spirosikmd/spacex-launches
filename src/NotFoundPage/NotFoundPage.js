import React from 'react';
import Image from '../Image';
import notFoundImage from './notFound.png';

const NotFoundPage = () => {
  return <Image alt="Not found" src={notFoundImage} />;
};

export default React.memo(NotFoundPage);
