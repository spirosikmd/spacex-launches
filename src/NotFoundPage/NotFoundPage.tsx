import React from 'react';
import Image from '../Image';
import notFoundImage from './notFound.png';
import { RouteComponentProps } from '@reach/router';

const NotFoundPage: (props: RouteComponentProps) => JSX.Element = () => {
  return <Image alt="Not found" src={notFoundImage} />;
};

export default React.memo(NotFoundPage);
