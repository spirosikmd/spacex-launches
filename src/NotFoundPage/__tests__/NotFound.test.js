import NotFoundPage from '../NotFoundPage';

describe('NotFoundPage', () => {
  it('renders', () => {
    expect(mountComponent(NotFoundPage, {})).toMatchSnapshot();
  });
});
