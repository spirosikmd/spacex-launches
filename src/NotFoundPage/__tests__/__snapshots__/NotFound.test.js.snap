// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage renders 1`] = `
<WithStyles(Image)
  alt="Not found"
  src="notFound.png"
>
  <div
    className="Image-imageWrapper-1"
  >
    <img
      alt="Not found"
      className="Image-image-2"
      src="notFound.png"
    />
  </div>
</WithStyles(Image)>
`;
